packages/ds-healthcare-gov/src/components/Header/MenuLinks.tsx

```diff
@@ -0,0 +1,51 @@
+import { LINK_IDENTIFIERS } from './defaultMenuLinks';
+import React from 'react';
+import { sendHeaderEvent } from './analytics';
+
+interface MenuLinksProps {
+  links: {
+    identifier?: string;
+    href: string;
+    label: React.ReactNode;
+    onClick?: (...args: any[]) => any;
```

Comment:
Same comment from Menu. There's probably a type that could be defined for reuse, but not sure if that's easy or worth right now